functions.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/** create a function and set fetch mode to connect to chair database */
+
+function connectdb()
+
+{
+    $db = new PDO('mysql:host=db; dbname=chairs', 'root', 'password');
+    $db ->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+/** create a function to display each chair and all their details from the database */
+
+function displayChairs($query): string
+{
+    $result = '';
+    foreach ($query as $chair) {
```

Comment:
What happens if there are no chairs in the database? Displaying an error message here could be a good idea!